runtime/src/system_instruction_processor.rs

```diff
@@ -116,7 +114,7 @@ fn finish_create_account(
         return Err(SystemError::InvalidAccountDataLength.into());
     }
 
-    // guard against sysvars being assigned
+    // guard against sysvars being made
```

Comment:
https://www.urbandictionary.com/define.php?term=made

Reply:
I think it's the first definition.